src/ensembl/src/content/app/browser/browserSelectors.ts

```diff
@@ -50,19 +50,19 @@ export const getBrowserNavStates = (state: RootState) =>
 export const getAllChrLocations = (state: RootState) =>
   state.browser.browserLocation.chrLocations;
 
-export const getChrLocation = (state: RootState) => {
+export const getChrLocation = (state: RootState): ChrLocation | null => {
   const chrLocations = getAllChrLocations(state);
   const activeGenomeId = getBrowserActiveGenomeId(state);
   return activeGenomeId ? chrLocations[activeGenomeId] : null;
 };
 
-export const getActualChrLocation = (state: RootState) => {
+export const getActualChrLocation = (state: RootState): ChrLocation | null => {
   const locations = state.browser.browserLocation.actualChrLocations;
   const activeGenomeId = getBrowserActiveGenomeId(state);
   return activeGenomeId ? locations[activeGenomeId] : null;
 };
 
-export const getDefaultChrLocation = (state: RootState) => {
+export const getDefaultChrLocation = (state: RootState): ChrLocation | null => {
   const activeEnsObjectId = getBrowserActiveEnsObjectId(state);
   const activeEnsObject =
     activeEnsObjectId && getEnsObjectById(state, activeEnsObjectId);
```

Comment:
Was typescript not smart enough to infer the return values for the three functions above itself?